--- ./Toggles.qml.orig	2026-08-07 08:51:53.448591011 -0300
+++ /etc/xdg/quickshell/caelestia/modules/utilities/cards/Toggles.qml	2026-08-07 08:52:05.319196934 -0300
@@ -132,6 +132,14 @@
                     }
                 }
                 DelegateChoice {
+                    roleValue: "blur"
+                    delegate: Toggle {
+                        icon: BlurMode.enabled ? "blur_on" : "blur_off"
+                        checked: BlurMode.enabled
+                        onClicked: BlurMode.enabled = !BlurMode.enabled
+                    }
+                }
+                DelegateChoice {
                     roleValue: "dnd"
                     delegate: Toggle {
                         icon: "notifications_off"
